homeassistant/components/temper/sensor.py

```diff
@@ -83,6 +86,7 @@ def __init__(self, temper_device, name, scaling):
         self.set_temper_device(temper_device)
 
         self._attr_name = name
+        self._attr_unique_id = f"temper_{temper_device.type.name}_temperature"
```

Comment:
This change seems to be unrelated to the PR?

Reply:
Since this now uses a configurable `suggested_display_precision` it looks like it needs a unique ID to allow it to be updated from the UI